src/ledger/trustlines.ts

```diff
@@ -31,7 +31,7 @@ async function getTrustlines(
   // 2. Make Request
   const responses = await this._requestAll('account_lines', {
     account: address,
-    ledger_index: await this.getLedgerVersion(),
+    ledger_index: options.ledgerVersion,
```

Comment:
I think we still want to default to `this.getLedgerVersion()` when no options is provided.

Can we do
```ts
const currentLedger = await this.getLedgerVersion()

const ledger_index = options.ledgerVersion ?? currentLedger

const responses = await this._requestAll('account_lines', {
    ........
    ledger_index,
    ........
}
```